test/lib/sun/hotspot/WhiteBox.java

```diff
@@ -592,7 +592,8 @@ public Object getMethodOption(Executable method, String name) {
   public native boolean isSharingEnabled();
   public native boolean isShared(Object o);
   public native boolean isSharedClass(Class<?> c);
-  public native boolean areSharedStringsIgnored();
+  public native boolean areSharedStringsMapped();
+  public native boolean isSharedInternedString(String s);
```

Comment:
please also update `jdk.test.whitebox.WhiteBox`

Reply:
I have synced jdk/test/whitebox/WhiteBox.java to match the latest APIs in sun/hotspot/WhiteBox.java